CONTRIBUTING.md

```diff
@@ -1,140 +1,296 @@
-# Contributing
+# Contributing to *PROJECT NAME*
 
-By contributing you agree to the [LICENSE](LICENSE) of this repository.
+*Note*: Be nice to your users and let them know you care about their
+contributions. Feel free to edit the text below.
```

Comment:
One the ideas we discussed was making this file quicker to use for most cases. I'd be interested to see what you think about dropping such instructions for project maintainers?

Reply:
Or perhaps having all instructions up the top so they can be deleted more easily?

Reply:
You're right. I moved all the instructions to the top to remove them easier. I left a note on the *Review and approval* step because it's empty and for me it's best to keep a note there for now.